plugins/poll/plugin.rb

```diff
@@ -230,6 +230,66 @@ def voters(post_id, poll_name, user, opts = {})
         serialized_voters(poll, opts)
       end
 
+      def transform_for_user_field_override(custom_user_field)
+        existing_field = UserField.find_by(name: custom_user_field)
+        existing_field ? "user_field_#{existing_field.id}" : custom_user_field
+      end
+
+      def grouped_poll_results(post_id, poll_name, user_field_name, user)
+        post = Post.find_by(id: post_id)
+        raise Discourse::InvalidParameters.new("post_id is invalid") unless post
```

Comment:
Usually we pass the symbol for the parameter to this exception. Ex: `post_id` (same note below.)